cmd/node/sync.go

```diff
@@ -0,0 +1,82 @@
+package node
+
+import (
+	"context"
+	"github.com/algorandfoundation/algorun-tui/api"
+	"github.com/algorandfoundation/algorun-tui/cmd/utils"
+	"github.com/algorandfoundation/algorun-tui/internal/algod"
+	"github.com/algorandfoundation/algorun-tui/ui/style"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// syncEndpoint is a string variable used to store the algod API endpoint address for communication with the node.
+var syncEndpoint string
+
+// syncToken is a string flag used to store the admin token required for authenticating with the Algod API.
+var syncToken string
+
+// defaultLag represents the default minimum catchup delay in milliseconds for the Fast Catchup process.
+var defaultLag int = 30_000
+
+var syncCmdShortTxt = "Quickly catch up your node to the latest block."
+var syncCmdLongTxt = lipgloss.JoinVertical(
+	lipgloss.Left,
+	style.Purple(style.BANNER),
+	"",
+	style.Bold(syncCmdShortTxt),
+	"",
+	style.BoldUnderline("Overview:"),
+	"Fetch the latest catchpoint and use Fast-Catchup to check if the node is caught up to the latest block.",
```

Comment:
This description indicates it is a status check (and not an action)

Reply:
This one is technically both, since it is using the `min` parameter it should exit early if is within the round window. It could have better verbiage to explain the early exit